src/schema/rules/sidecars/continuous.yaml

```diff
@@ -19,7 +19,51 @@ PhysioHardware:
   selectors:
     - suffix == "physio"
   fields:
+    DeviceSerialNumber: recommended
     Manufacturer: recommended
     ManufacturersModelName: recommended
     SoftwareVersions: recommended
-    DeviceSerialNumber: recommended
+
+PhysioTypeRecommended:
+  selectors:
+    - suffix == "physio"
+  fields:
+    PhysioType: recommended
+
+PhysioTypeRequired:
+  selectors:
+    - suffix == "physio"
+  fields:
+    PhysioType: required
+
+PhysioEvents:
+  selectors:
+    - suffix == "physioevents"
+  fields:
+    Columns: required
+    Description: recommended
+    ForeignIndexColumn: recommended
+
+# Eye-tracking
+EyeTrack:
+  selectors:
+    - suffix == "physio"
+  fields:
+    EnvironmentCoordinates: required
+    RecordedEye: required
+    SampleCoordinateUnits: required
```

Comment:
Note that the unit may also described in the description of each column of the physio file so so this may conflict with also having it here, no?